core-it-support/core-it-plugins/maven-it-plugin-configuration/src/main/java/org/apache/maven/plugin/coreit/ConfigMojo.java

```diff
@@ -118,7 +118,7 @@
     /**
      * A simple parameter of type {@link java.lang.Integer}.
      *
-     * @parameter property="config.intergerParam"
+     * @parameter property="config.integerParam"
```

Comment:
Attention: Is this property used in the IT?

Reply:
It did not find any reference in this repo. And according to https://github.com/search?q=config.intergerParam&type=code there is no references in other repos either.

Do you think it's safer to roll back the change ?

Reply:
Use grep, then we are sure.

Reply:
No result when grepping on `intergerParam`.